code/modules/client/client procs.dm

```diff
@@ -176,7 +176,10 @@ var/next_external_rsc = 0
 
 	while (query.NextRow())
 		player_age = text2num(query.item[2])
-		break
+		return
+	//player not found.
+	player_age = 0

```

Comment:
Could we change this to player_age < [something like 20 or 30], then send a message like "[player] is connecting here for the [player_age]th time"? I'd kinda like to have it where it tells us up to a certain number, and not just the first time they're connecting.